cli/command/formatter/licenses.go

```diff
@@ -0,0 +1,103 @@
+package formatter
+
+import (
+	"time"
+
+	"github.com/docker/cli/cli/licenseutils"
+)
+
+// TODO - derived from plugin.go - replicate plugin_tests.go before PR
+
+const (
+	defaultSubscriptionsTableFormat = "table {{.Num}}\t{{.Owner}}\t{{.ProductID}}\t{{.Expires}}\t{{.Components}}"
+	defaultSubscriptionsQuietFormat = "{{.Num}}:{{.ProductID}}"
+
+	numHeader                 = "NUM"
+	licenseSubscriptionHeader = "SUBSCRIPTION"
+	stateHeader               = "STATE"
+	productIDHeader           = "PRODUCTID"
```

Comment:
[Other headers](https://github.com/docker/cli/blob/master/cli/command/formatter/task.go#L20) seems to separate words with a space. So it should be `PRODUCT ID` and "RATE PLAN" ?